test/unit/voldemort/store/rocksdb/RocksdbStorageEngineTest.java

```diff
@@ -85,11 +83,9 @@ public void tearDown() throws Exception {
     @Override
     public void testTruncate() throws Exception {}
 
-    @Ignore
```

Comment:
I think here you need to completely remove the override, otherwise, you are still overriding that particular test with an empty implementation.